magefiles/scala.go

```diff
@@ -32,5 +32,6 @@ func BuildScala() error {
 		"--rm",
 		"-v", fmt.Sprintf("%s:/build", wd),
 		"-w", "/build",
+		"--env", "HOME=/tmp",
```

Comment:
Running as part of the Github Actions CI, default `HOME` will point to a location  (`/`) where the user (probably the test runner user)  running the build script does not have write permissions. This home works for any user running the script in the Docker container.